doc/api/process.md

```diff
@@ -2601,6 +2601,77 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.permission`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+
+This API is available through the [`--experimental-permission`][] flag.
+
+`process.permission` is an object whose methods are used to manage permissions
+for the current process. Additional documentation is available in the
+[Permission Model][].
+
+### `process.permission.check(scope[, reference])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `scopes` {string}
+* `reference` {string}
+* Returns: {boolean}
+
+Verifies that the process is able to access the given scope and reference.
+If no reference is provided, a global scope is assumed.
+
+The reference has a meaning based on the provided resource. For example,
```

Comment:
I don't understand what this first sentence means. Should "resource" be "scope"?

Reply:
Yes, thanks